<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormContact</class>
 <widget class="QWidget" name="FormContact">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FormContact</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Tinos</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#a51d2d;&quot;&gt;Gestion du contact sélectionné&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vous souhaitez : </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonAfficher">
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonGerer">
       <property name="text">
        <string>Gérer</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonModifier">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAjouteInter">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ajouter Intéraction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSupprimer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Supprimer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonAnnuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>FormContact</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
